curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f8f8bcd4370f6509078e.md

```diff
@@ -16,6 +16,8 @@ California<br />
 USA
 ```
 
+The `br` tags will allow each part of the address to be on their own line and are 
+essential in presenting `address` elements properly. 
```

Comment:
"Essential" is a pretty strong word here. Do you have a source you can site for this?

Reply:
```suggestion
The `br` tags will allow each part of the address to be on their own line and are 
useful for presenting `address` elements properly. 
```

I'd go with MDN's "useful". 

> It is useful for writing a poem or an address, where the division of lines is significant.

Reply:
Good change @naomi-lgbt. Since this is a unit about accessibility I was just worried that using "essential" made it appear that you must always use `br` tags in the `address` element because it helps with accessibility. For the record, they do not.